docs/templates/reachview/firmware-reflashing.mdx

```diff
@@ -187,10 +187,21 @@ Get the latest ReachView image version and unzip it:
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/common/app-step2-choose-usb-rs.png" style="width: 300px;" /></p>
 {%endif%}
 
-* Select ReachView firmware .img file and click on _Flash_
+* Download the proposed ReachView firmware .zip file. You can also select .zip or .img file from your disk. If your image file is outdated, the Flasher tool will show you a warning message. Click on _Flash_
```

Comment:
You can also select .zip or .img file from your disk. => Let's rephrase it a bit. Let's mention that the user can either unzip the image themself and select a .img file or just select a .zip file (in this case, flasher unzip it)

Reply:
What do you think about adding this as a note before the screenshot?

Reply:
Yep, let's do it